### PR TITLE
[AzureActiveDirectoryB2C] add test case and examples.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

add test case and examples.

### CODE REVIEW

File: src/ActiveDirectoryB2C/readme.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ This directory contains the PowerShell module for the ActiveDirectoryB2C service
 This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
 
 ## Module Requirements
-- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.2.3 or greater
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 1.8.1 or greater
```
Comment (BethanyZhou) [inline src/ActiveDirectoryB2C/readme.md:20]: Please regenerate using autorest --use:@autorest/powershell@3.0.414, thanks.

File: src/ActiveDirectoryB2C/docs/New-AzADB2CTenant.md [hunk 2]
```diff
@@ -23,23 +23,16 @@ Initiates an async request to create both the Azure AD B2C tenant and the corres
...
-### Example 1: {{ Add title here }}
+### Example 1: Create a tenant of the AzureActiveDirectory by name 
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> New-AzADB2CTenant -ResourceGroupName lucas-rg-test -Name asdsdsadsad.onmicrosoft.com -Location 'United States' -Sku Standard -CountryCode US -DisplayName "azure.onmicrosoft.com"
 
-{{ Add output here }}
+Location Name                                 Type
+-------- ----                                 ----
+         d005debc-8b64-4008-829c-e68f7d9349ab
```
Comment (BethanyZhou) [inline src/ActiveDirectoryB2C/docs/New-AzADB2CTenant.md:32]: why this name is a guid

File: src/ActiveDirectoryB2C/docs/Get-AzADB2CTenant.md [hunk 3]
```diff
@@ -39,23 +39,52 @@ Get the Azure AD B2C tenant resource.
...
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> Get-AzADB2CTenant
 
-{{ Add output here }}
+Location      Name                            Type
+--------      ----                            ----
+United States klaskkdls.onmicrosoft.com       Microsoft.AzureActiveDirectory/b2cDirectories
+United States asdsdsadsad.onmicrosoft.com     Microsoft.AzureActiveDirectory/b2cDirectories
+United States yishiTestTenant.onmicrosoft.com Microsoft.AzureActiveDirectory/b2cDirectories
```
Comment (BethanyZhou) [inline src/ActiveDirectoryB2C/docs/Get-AzADB2CTenant.md:50]: Please remove these information containing personal information.

File: src/ActiveDirectoryB2C/readme.md (hunk 1 above)
Comment (LucasYao93) [reply src/ActiveDirectoryB2C/readme.md]: Done

File: src/ActiveDirectoryB2C/docs/Get-AzADB2CTenant.md (hunk 3 above)
Comment (LucasYao93) [reply src/ActiveDirectoryB2C/docs/Get-AzADB2CTenant.md]: Done

File: src/ActiveDirectoryB2C/docs/New-AzADB2CTenant.md (hunk 2 above)
Comment (LucasYao93) [reply src/ActiveDirectoryB2C/docs/New-AzADB2CTenant.md:32]: The server has issue.

**Comment by BethanyZhou** [comment]
Waiting for swagger bug fix to re-record test case.

**Pull request closed without merging**